src/IdeaRS.OpenModel/Libraries/Material/MaterialBoltGrade.cs

```diff
@@ -10,6 +11,13 @@ namespace IdeaRS.OpenModel.Material
 	[OpenModelClass("CI.StructModel.Libraries.Material.MaterialBoltGrade,CI.Material", "CI.StructModel.Libraries.Material.IMatBoltGrade,CI.BasicTypes", typeof(Material))]
 	[DataContract]
 	[JsonObject(MemberSerialization = MemberSerialization.OptOut)]
+	[XmlInclude(typeof(MaterialBoltGradeAISC))]
+	[XmlInclude(typeof(MaterialBoltGradeAUS))]
+	[XmlInclude(typeof(MaterialBoltGradeCISC))]
+	[XmlInclude(typeof(MaterialBoltGradeEc2))]
+	[XmlInclude(typeof(MaterialBoltGradeHKG))]
+	[XmlInclude(typeof(MaterialBoltGradeCHN))]
+	[XmlInclude(typeof(MaterialBoltGradeIND))]
 	public class MaterialBoltGrade : Material
```

Comment:
should this be abstract now?

Reply:
Im not sure this need to be tested during implementation of import.